pom.xml

```diff
@@ -47,7 +47,6 @@
         <jafama.version>2.3.1</jafama.version>
         <junit-jupiter.version>5.5.2</junit-jupiter.version>
         <slf4jtoys.version>1.6.2</slf4jtoys.version>
-
```

Comment:
I think you have to leave the space here (it is a detail I know, it is for readability).